docs/rst/manual/conf.py

```diff
@@ -47,26 +47,19 @@
     "sphinx.ext.mathjax",
     "sphinx.ext.ifconfig",
     "sphinx.ext.githubpages",
+    "sphinx.ext.autodoc",
     "sphinx.ext.napoleon",
     "sphinxarg.ext",
     "sphinx.ext.todo",
     "sphinxcontrib.datatemplates",
     "ert_jobs",
     "ert_narratives",
-    "autoapi.extension",
 ]
-autoapi_type = "python"
-autoapi_dirs = ["../../../ert3/workspace"]
-autoapi_add_toctree_entry = False
-# Options: https://sphinx-autoapi.readthedocs.io/en/latest/reference/config.html#customisation-options
-autoapi_options = [
-    "members",
-    "inherited-members",
-    "private-members",
-    "special-members",
-    "show-inheritance",
-]
-autoapi_python_class_content = "both"
+
+# Autodoc settings:
+autodoc_class_signature = "separated"
+autodoc_inherit_docstrings = False
+nitpick_ignore = [("py:class", "pydantic.types.FilePath")]
```

Comment:
Do we lose the recursive feature of autoapi?
Initially I tried extending autodoc [like suggested in this SO answer](https://stackoverflow.com/a/62613202/7456798), but could not get it to work.

Reply:
I did not look into this because right now we want only to document the workspace module. If we are going to have more extensive docs, we should revisit that.